tests/test_chatpermissions.py

```diff
@@ -124,3 +124,15 @@ def test_equality(self):
 
         assert a != d
         assert hash(a) != hash(d)
+
+    def test_all_true(self):
+        f = ChatPermissions()
+        t = ChatPermissions.all_true()
+        # if the dirs are the same, the attributes will all be there
+        assert dir(f) == dir(t)
+        # now we just need to check that all attributes are True. _id_attrs returns all values,
+        # if a new one is added without defaulting to True, this will fail
+        for key in t._id_attrs:
```

Comment:
imo we should test with `.__slots__` since we won't update `_id_attrs` when a new attribute is added

Reply:
Ah, backwards compatibility? Or whats the reason for not updating.

Reply:
yep backwards compatibility

Reply:
yay backwards compatibility